packages/api-explorer/src/CodeSample.jsx

```diff
@@ -92,11 +90,7 @@ class CodeSample extends React.Component {
               </ul>
               <div className="hub-code-auto">
                 <CopyCode code={code} />
-                <pre
-                  className={`tomorrow-night hub-lang hub-lang-${language}`}
-                  // eslint-disable-next-line react/no-danger
-                  dangerouslySetInnerHTML={{ __html: snippet }}
-                />
+                <pre className={`tomorrow-night hub-lang hub-lang-${language}`}>{snippet}</pre>
```

Comment:
Why does snippet no longer need to be set with `dangerouslySetInnerHTML`

Reply:
The syntax-highlighter returns react elements now! Because we have to be able to swap out variables in the middle of code blocks: https://github.com/readmeio/api-explorer/blob/3147975c4c52d914ce7964faaa5e2ed0b6ddff0d/packages/syntax-highlighter/index.js